src/Components/Admin/Assets/assetCard.js

```diff
@@ -79,7 +79,12 @@ export default function AssetCard({ data, handleDeleteClick }) {
             >
                 <img
                     className={classes.cardImage}
```

Comment:
Given the different heights/widths of the default image and of the screenshot, let's add an attribute to cardImage.

If you add: `objectFit: contains` or `objectFit: covers` (please test which one looks better locally). You can find more about object-fit property here: https://www.w3schools.com/css/css3_object-fit.asp

Can you also retroactively add this single prop to the other cards (scenarioCard and sceneCard)? Thanks :)

Reply:
Wanted to add an image to show you what I mean. The armchair is too tall, so using object-fit should make it look better. 
![image](https://user-images.githubusercontent.com/13268990/125172604-95cd6580-e188-11eb-80f2-f78ad70b7b92.png)